File: Euler_002/Euler_2.java
class Euler_2{
  public static void main(String[] args) {
    int c = 0;
    int sum = 0;
    int[] i = {1,1};

    while (c < 4000000){
      c = i[0] + i[1];
      i[0] = i[1];
      i[1] = c;
      if (c % 2 == 0){
        sum += c;
      }
    }
    System.out.println(sum);
  }
}

File: Euler_002/Euler_2.swift
let four_mil: Int = 4000000
var i:  [Int] = [1,1]
var c: Int = 0
var sum: Int = 0

while c < four_mil{
	c = i[0] + i[1]
	i[0] = i[1]
	i[1] = c
	if c % 2 == 0{
		sum += c
	}
}
print(sum)
